ui/src/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -8690,12 +8690,20 @@ public void addJobGraph( JobMeta jobMeta ) {
     delegates.jobs.addJobGraph( jobMeta );
   }
 
+  public boolean addSpoonBrowser( String name, String urlString, LocationListener locationListener, boolean toolbar ) {
```

Comment:
Use a "predicate" argument name for boolean arguments instead of a noun (toolbar). I suggest using "showControls" which is the name of the argument in the SpoonBrowser class.